lua/lualine/extensions/chadtree.lua

```diff
@@ -0,0 +1,13 @@
+-- Copyright (c) 2020-2021 hoob3rt
+-- MIT license, see LICENSE for more details.
+local nerdtree = require('lualine.extensions.nerdtree')
+
+local M = {}
+
+M.sections = nerdtree.sections
+
+M.inactive_sections = M.inactive_sections
```

Comment:
Should be `M.sections`

Reply:
actually it should be `nerdtree.inactive_sections`. Idk how I have not noticed, thanks :smile:

Reply:
Right nerdtree.inactive_section. 

Though I'm still in favour of having inactive_section permannently linked to sections in extentions as all extention do it anyway